sdk/core/Azure.Core.TestFramework/src/RecordedTestBase.cs

```diff
@@ -50,6 +51,36 @@ private string GetSessionFilePath(string name = null)
             return Path.Combine(TestContext.CurrentContext.TestDirectory, "SessionRecords", className, fileName);
         }
 
+        /// <summary>
+        /// Add a static TestEventListener which will redirect SDK logging
+        /// to Console.Out for easy debugging.
+        /// </summary>
+        private static TestLogger Logger { get; set; }
+
+        /// <summary>
+        /// Start logging events to the console if debugging or in Live mode.
+        /// This will run once before any tests.
+        /// </summary>
+        [OneTimeSetUp]
+        public void StartLoggingEvents()
+        {
+            if (Debugger.IsAttached || Mode == RecordedTestMode.Live)
```

Comment:
Nit - flip the order of the two conditions.  `Debugger.IsAttached` is probably a lot more expensive to check.